CMakeLists.txt

```diff
@@ -139,12 +132,14 @@ set(TARGET_LIB_DIR "${AWS_LIBRARY_OUTPUT_DIR}/${TARGET_OS}/${TARGET_ARCH}")
 
 # shared lib that contains the CRT and JNI bindings, to be loaded by java
 add_library(${PROJECT_NAME} SHARED ${CRT_JAVA_HEADERS} ${CRT_JAVA_SRC})
+if (UNIX AND NOT APPLE)
+    aws_use_package(s2n)
```

Comment:
how on earth was s2n building without this before?

Reply:
![](https://i.ytimg.com/vi/F3P5ABfwvuo/maxresdefault.jpg)

Reply:
Oh actually, this probably isn't necessary now. It worked previously because aws-c-io does aws_use_package(s2n).

Lemme back this sucker out and make sure.